guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb

```diff
@@ -0,0 +1,1555 @@
+{
```

Comment:
<p>"However, we recommend using <code>arcpy</code> for better accuracy <strong>and support for a wider gamut of data sources"</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>